### PR TITLE
Feature/spectron-left-panels

### PR DESCRIPTION
**Pull request by Jess-Jian**
* rework ellipse-test.js file
* rework line-test.js file
* rework pen-test.js file
* rework polygon-test.js file

### CODE REVIEW

File: test/features/editor/left-panels-test/ellipse-test.js [hunk 1]
```diff
@@ -2,42 +2,44 @@ const { checkExist, setReload } = require('../../../util/utils');
...
+    { type: 'pointerMove', x: 400, y: 400, duration: 100, },
+    { type: 'pointerDown', button: 0, },
+    { type: 'pointerMove', x: 464, y: 464, duration: 1000, },
+    { type: 'pointerUp', button: 0, },
+  ]);
+  await checkExist('#svg_1');
+
+  const startPoint = await pageCoordtoCanvasCoord({ x: 400, y: 400 });
+  const endPoint = await pageCoordtoCanvasCoord({ x: 464, y: 464 });
+  let expectedX = startPoint.x;
```
Comment (dkdkwizard) [inline test/features/editor/left-panels-test/ellipse-test.js:23]: use `const` if you won't change it.

File: test/features/editor/left-panels-test/line-test.js [hunk 2]
```diff
@@ -2,54 +2,53 @@ const { checkExist, setReload } = require('../../../util/utils');
...
+    { type: 'pointerMove', x: 300, y: 300, duration: 100, },
+    { type: 'pointerDown', button: 0, },
+    { type: 'pointerMove', x: 500, y: 500, duration: 1000, },
+    { type: 'pointerUp', button: 0, },
+  ]);
+  await checkExist('#svg_1');
+
+  const startPoint = await pageCoordtoCanvasCoord({ x: 300, y: 300 });
+  const endPoint = await pageCoordtoCanvasCoord({ x: 500, y: 500 });
+  let expectedX1 = startPoint.x;
```
Comment (dkdkwizard) [inline test/features/editor/left-panels-test/line-test.js:23]: use `const` if you won't change it.

File: test/features/editor/left-panels-test/ellipse-test.js (hunk 1 above)
Comment (Jess-Jian) [reply test/features/editor/left-panels-test/ellipse-test.js:23]: updated

File: test/features/editor/left-panels-test/line-test.js (hunk 2 above)
Comment (Jess-Jian) [reply test/features/editor/left-panels-test/line-test.js]: updated

**Review by dkdkwizard — APPROVED** [review]

**Pull request merged**